src/ProtectedProxies/ProtectedMinimalProxy.sol

```diff
@@ -11,15 +11,16 @@ import {SphereXProtectedProxy} from "../SphereXProtectedProxy.sol";
 contract SpherexProtetedMinimalProxy is SphereXProtectedProxy {
```

Comment:
What's the naming decision? ProtectedMinimalProxy or MinimalProtectedProxy?

Reply:
will fix it